app/models/user_history.rb

```diff
@@ -205,7 +207,9 @@ def self.staff_actions
       :topic_archived,
       :topic_unarchived,
       :post_staff_note_create,
-      :post_staff_note_destroy
+      :post_staff_note_destroy,
+      :add_watched_word,
+      :delete_watched_word
```

Comment:
For consistency with other actions, maybe `watched_word_create` and `watched_word_destroy`.